modules/microshift-cni-customization-matrix.adoc

```diff
@@ -0,0 +1,47 @@
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-nw-customization-matrix_{context}"]
+= {microshift-short} networking customization matrix
+
+The following table summarizes the status of networking features and capabilities that are either present as defaults, supported for configuration, or not available with the {microshift-short} service:
+
+.{microshift-short} networking capabilities and customization status
+[cols="50%,20%,30%",options="header"]
+|===
+|Network feature|Availability|Customization supported
+
+|Advertise address|Yes|Yes ^[1]^
+
+|Kubernetes network policy|Yes|Yes
+
+|Kubernetes network policy logs|Not available|N/A
+
+|Load balancing|Yes|Yes
```

Comment:
This looks unclear to me. Does it mean the support to the LoadBalancer type of k8s Services?